upcloud/resource_upcloud_objectstorage.go

```diff
@@ -17,6 +22,12 @@ import (
 
 const bucketKey = "bucket"
 const numRetries = 5
+const AccessKeyEnvVarPrefix = "UPCLOUD_OBJECT_STORAGE_ACCESS_KEY_"
+const SecretKeyEnvVarPrefix = "UPCLOUD_OBJECT_STORAGE_SECRET_KEY_"
```

Comment:
I think that exporting these at the moment is not necessary, Our tests lives also inside `upcloud` package.